roles/network_plugin/canal/templates/network.json

```diff
@@ -0,0 +1 @@
+{ "Network": "{{ kube_pods_subnet }}", "SubnetLen": {{ kube_network_node_prefix }}, "Backend": { "Type": "{{ flannel_backend_type }}" } }
```

Comment:
Also, kube_pods_subnet and kube_network_node_prefix are missing defaults, is it ok?